src/RVRP_data.jl

```diff
@@ -63,7 +63,8 @@ end
 
 mutable struct VehicleCategory
     id::String
-    compartment_capacities::Array{Float64,2} # matrix providing capacites for each compartment the additive measure: weight, value, volume
+    compartment_capacities::Array{Float64,2} # matrix providing capacites for each compartment the additive measures: weight, value, volume
+    compartment_properties::Array{Float64,2} # matrix providing capacites for each compartment the properties to check: max weight, max heid, capacity of carrying liquid, Refrigerated product, ...
```

Comment:
I think something is missing in this sentence : `# matrix providing capacites for each compartment the properties to check`